src/app/ApplicationTemplate.UWP/Views/Styles/Controls/CommandBar.xaml

```diff
@@ -82,7 +82,8 @@
 			<Setter.Value>
 				<DataTemplate>
 					<TextBlock Text="{Binding}"
-							   Style="{StaticResource Headline6}" />
+							   Style="{StaticResource Headline6}"
+							   Foreground="{ThemeResource MaterialSurfaceBrush}" />
```

Comment:
It doesn't make sense to use **Surface** for a **foreground** brush.

Reply:
I've set it to MaterialOnPrimaryColor now